documents/WCA Bylaws.md

```diff
@@ -59,27 +97,30 @@ The number of authorized directors of the WCA shall be not less than three (3) n
 ### 3.3 Qualification of Directors
 No more than forty-nine percent (49%) of the directors serving on the Board may be interested persons, as defined in Section 5227 of the Nonprofit Corporation Law. However, any violation of this Section 3.3 shall not affect the validity or enforceability of any transaction entered into by the WCA.
 
-### 3.4 Election and Term of Office
+### 3.4 Eligibility of Directors
+To be eligible as a director of the WCA candidates must be at least 18 years of age and be a voting member of the WCA.
```

Comment:
The Motion 05.2019.1 states:
> 3. All the WCA Members with voting rights at the start of the voting are eligible to vote.

So this Motion should be updated.

Reply:
No change needed. Point 3 only specifies who is eligible to vote in the election not who is eligible to run